geode-core/src/test/java/org/apache/geode/management/internal/beans/DistributedSystemBridgeJUnitTest.java

```diff
@@ -51,13 +53,12 @@
 import org.apache.geode.test.fake.Fakes;
 
 public class DistributedSystemBridgeJUnitTest {
+  @Rule
+  public TemporaryFolder temporaryFolder = new TemporaryFolder();
```

Comment:
Please rename the test to DistributedSystemBridgeIntegrationTest and move it from geode-core/src/test to geode-core/src/integrationTest.

Reply:
Hey @kirklund,

Why should I categorise this as an `integrationTest` if it only uses fakes and mocks?.

Reply:
Disregard that, it uses the file system through the `TemporaryFolder`, got it.